components/fxa-client/src/http_client.rs

```diff
@@ -131,6 +137,19 @@ impl FxAClient for Client {
         self.make_oauth_token_request(config, body)
     }
 
+    fn oauth_introspection_with_refresh_token(  // TODO
```

Comment:
You can remove the `_with_refresh_token` suffix as there's only one version.

Reply:
To be fair, you could quite legitimately implement matching `introspect_refresh_token` and `introspect_access_token` methods, although we don't currently have any use for the later.